frontend/contact/index.html

```diff
@@ -125,8 +123,23 @@ <h3>My domain (or one of its subdomains) is broken due to preloading, and waitin
 
   <section id="other">
     <h2><a class="hash-link" href="#other">Other questions</a></h2>
```

Comment:
nit: I don't like that these are the only questions that aren't phrased as a question, but I couldn't figure out how to rephrase them

Reply:
Could cheat by ending each sentence with: "; can you help?" 😛